src/components/ContactForm/ContactForm.js

```diff
@@ -12,22 +15,22 @@ const ContactForm = () => {
     'En validant, vous autorisez HOFMANN.AI à stocker les données personnelles soumises afin qu\'elles fournissent le contenu souhaité.'
   );
   const [contactCheck, setContactCheck] = useState('');
-
+  if (url === null || url === undefined) return <></>;
```

Comment:
Should be `if (!url) return <></>;`

Reply:
We tester it out with Jacques, and it did not pass the linter.